reproducibility_intro_activity.Rmd

```diff
@@ -72,78 +77,85 @@ We will practice filing a pull request by filling in the the README. READMEs are
-<input type="checkbox"> Click `Create pull request`.  
 
+<input type="checkbox"> Click the green `Create pull request` button.  
 
 ## Activity 2: Navigating PR components
 
 ### 2a: Leaving comments and suggestions
 
-<input type="checkbox"> Return to your pull request from Activity 1.   
-<input type="checkbox"> Click on the `Files Changed` tab.   
-<input type="checkbox"> Click one of the plus signs to the left to leave a comment.   
-<input type="checkbox"> With one of the open comments click the little symbol that has a plus and minus in a page to leave a suggestion.  
-<input type="checkbox"> Click `Add a single comment`.  
+<input type="checkbox"> Return to your pull request on GitHub in your browser from Activity 1.   
+<input type="checkbox"> Click on the `Files Changed` tab (lower set of tabs).   
+<input type="checkbox"> Click one of the plus signs to the left of one of your changes to leave a comment. Type something and then click `Add single comment`. Nice, you added a comment!
+<input type="checkbox"> Click another plus sign to the left of a change.
+<input type="checkbox"> This time, click the little symbol that has a plus and minus in a page (next to the preview tab). This will start a suggestion, where you can make suggested changes to documents. 
+<input type="checkbox"> Make a change to the text in the suggestion.  
+<input type="checkbox"> Click `Add a single comment`. Great, You added a suggestion!
 
 Note that you or a collaborator can now readily incorporate your suggestion by clicking `Commit suggestion` or they could comment back to further discuss.
 
 <input type="checkbox"> Click `Commit suggestion` and then `Commit changes`.  
-<input type="checkbox"> Return to the `Conversation` tab and scroll to the bottom. You should see an additional commit now!
-
-### 2b: Reading GitHub Actions logs
+<input type="checkbox"> Return to the `Conversation` tab and scroll to the bottom. You should see an additional commit and your comment now!
 
-On the bottom of your PR you will notice a box at the bottom that will probably either say `Some checks haven't completed yet` or `All checks passed`.  
-
-This is an automatic check run by GitHub Actions which is one of many systems used for Continuous Integration/Continuous Deployment.
-
-<input type="checkbox"> If needed, click `Show all checks`.  
-<input type="checkbox"> Click `Details` at the end of the line that says `Render R Example / render-notebooks (pull_request)`.  
-<input type="checkbox"> Click around in here to investigate what GitHub Actions logs look like.
-
-For more learning about using [GitHub Actions for science see our course](https://hutchdatascience.org/GitHub_Automation_for_Scientists/).
-
-### 2c: Updating your local repository
+### 2b: Updating your local repository
```

Comment:
2b and 2c were in the opposite order in the slides - I think the slide version makes more sense